data/schema/filters/weapon.cfg

```diff
@@ -15,3 +15,6 @@
 	{SIMPLE_KEY movement_used s_range_list}
 	{FILTER_BOOLEAN_OPS weapon}
 [/tag]
+
+{FILTER_TAG "$filter_second_weapon" weapon ()}
```

Comment:
Um, I meant using this at the point of use (which you're doing), not here in weapon.cfg. You don't really need it here.

Reply:
Ah, alright.